invader_payment_sips/services/payment_sips.py

```diff
@@ -59,21 +58,17 @@ class PaymentServiceSips(AbstractComponent):
     _usage = "payment_sips"
     _description = "REST Services for SIPS payments"
 
-    @property
-    def payment_service(self):
-        return self.component(usage="invader.payment")
-
     def _validator_prepare_payment(self):
         schema = {
-            "payment_mode_id": {
-                "coerce": to_int,
-                "type": "integer",
-                "required": True,
-            },
+            "payment_mode": {"type": "string"},
```

Comment:
I thinks there is an issue during the rebase.
it should be payment_mode_id and type int